src/components/Records/SearchFilters.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="container">
+    <form class="card" v-if="filters.length > 0">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['v-if' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4961435)